fhirflat/resources/extensions.py

```diff
@@ -417,6 +417,42 @@ def elements_sequence(cls):
         ]
 
 
+class Race(_DataType):
+    """
+    An ISARIC extension collecting data on the race of a patient.
+    """
+
+    resource_type = Field("Race", const=True)
```

Comment:
should be "race"

Reply:
The url (so the bit used in the the FHIR json file) is lower case, but I've kept one-word classes as capitalised to keep the code more pythonic and easier to read. Same with Age, Duration etc. Once it gets into FHIR, everything is lower-case as required by their docs.